docs: landing and architecture page updates (#26)
armory/docs-cdaas: content/en/_index.md

  type: "home"
no_list: true
description: >

>   A declarative developer experience that represents how engineers actually want their software deployed. Easily integrate with the tools you already use via a flexible CLI and custom web-hooks. No more need for custom scripts and duct-taped solutions.

Reviewer: ```suggestion
Armory CD-as-a-Service is a declarative deployment experience that represents how engineers actually want their software deployed. Easily integrate with the tools you already use via a flexible CLI and custom webhooks. 
```

Shortened but still over the 160 character limit

---

> ## What Continuous Deployment-as-a-Service is

Reviewer: ```suggestion
## What CD-as-a-Service is
```

> Armory Continuous Deployment-as-a-Service is a centralized control plane that allows you to deploy to multiple Kubernetes clusters using our secure agent architecture. CD-as-a-Service enables multi-environment, multi-region application rollouts with promotion constraints and advanced deployment strategies, such as canary and blue/green.

Reviewer: ```suggestion
CD-as-a-Service is a centralized control plane that enables deployment to multiple Kubernetes clusters using a secure Kubernetes agent architecture. CD-as-a-Service supports multi-environment, multi-region app deployments with promotion constraints and advanced deployment strategies, such as canary and blue/green.
```

Changed "allows" to "enables" because CDaaS is all about empowerment

> ## Why Armory CD-as-a-Service

Reviewer: ```suggestion
## Why use CD-as-a-Service
```

> Modern software delivery requires sophisticated control of delivery speed, traffic management, and integration with external tools and verification processes. 

Reviewer: ```suggestion
Modern software delivery requires sophisticated control of delivery speed and traffic management, as well as integration with external tools and verification processes. 
```
nit: thought there were too many "and"

> Software Delivery is rarely a single environment. Most workflows require promotion from one environment or region to another after some set of constraints has been met, such as test executions, manual approvals, CI workflows, canary analysis, etc.

Reviewer: ```suggestion
Software is rarely deployed to a single environment. Most workflows require promotion from one environment or region to another after meeting constraints such as test executions, manual approvals, CI workflows, or canary analysis.
```
